src/wallet/components/WalletDropdownFundLink.tsx

```diff
@@ -1,34 +1,79 @@
+import { useCallback, useMemo } from 'react';
 import { useGetFundingUrl } from '../../fund/hooks/useGetFundingUrl';
+import { getPopupSize } from '../../fund/utils/getPopupSize';
+import { openPopup } from '../../internal/utils/openPopup';
+import { cn, pressable, text as themeText } from '../../styles/theme';
+import { useIcon } from '../hooks/useIcon';
 import type { WalletDropdownFundLinkReact } from '../types';
-import { WalletDropdownFundLinkButton } from './WalletDropdownFundLinkButton';
 
 export function WalletDropdownFundLink({
```

Comment:
Ended up merging this back into a single component, you guys were right @0xAlec @alessey